content/en/docs/concepts/architecture/nodes.md

```diff
@@ -392,6 +392,50 @@ Failed pod objects will be preserved until explicitly deleted or [cleaned up by
 This is a change of behavior compared to abrupt node termination.
 {{< /note >}}
 
+## Swap memory management {#swap-memory}
+
+{{< feature-state state="alpha" for_k8s_version="v1.22" >}}
+
+Prior to Kubernetes 1.22, nodes did not support the use of swap memory, and a
+kubelet would fail to start if swap was detected on a node by default. In 1.22
+onwards, swap memory support can be configured on a per-node basis.
+
+To enable swap on a node, the `NodeSwap` feature flag must be set to true on
+the kubelet, and the `--fail-swap-on` command line flag or `failSwapOn`
+[configuration setting] must be set to false.
+
+[configuration setting]: https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration
+
+A user can also optionally configure `memorySwap.swapBehavior` in order to
+configure how the node will use swap memory. For example,
+
+```yaml
+memorySwap:
+  swapBehavior: LimitedSwap
+```
+
+The available configuration options for `SwapBehavior` are:
+
+- Unset: defaults to `LimitedSwap`
```

Comment:
Maybe we want to remove this line or clarify what "Unset" means. Does it mean that the "swapBehavior" key is omitted or the "swapBehavior" is set to `null`, or both?

Reply:
Both.

Reply:
Moved this to a trailing paragraph.